google/cloud/dataproc_spark_connect/session.py

```diff
@@ -24,8 +24,10 @@
 import time
 import uuid
 import tqdm
+from tqdm import tqdm as cli_tqdm
+from tqdm.notebook import tqdm as notebook_tqdm
```

Comment:
I thought that tqdm will detect whether it runs in CLI automatically and have text progress (at least for session creation it seems to work)?